src/common/choiccmn.cpp

```diff
@@ -73,8 +73,10 @@ wxEVENT_PROPERTY( Select, wxEVT_CHOICE, wxCommandEvent )
 
 wxPROPERTY( Font, wxFont, SetFont, GetFont , wxEMPTY_PARAMETER_VALUE, \
            0 /*flags*/, wxT("Helpstring"), wxT("group"))
+#if 0
```

Comment:
Why leave it here at all? I.e. why not just remove it?

P.S. Same question for all `#if 0`s below too.